src/index.html

```diff
@@ -1,14 +1,33 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="snake">
+      <div class="block__one">
+        <span class="block__number">1</span>
+      </div>
+      <div class="block__two">
+        <span class="block__number">2</span>
+      </div>
+      <div class="block__three">
+        <span class="block__number">3</span>
+      </div>
+      <div class="block__four">
+        <span class="block__number">4</span>
+      </div>
+      <div class="block__five">
+        <span class="block__number">5</span>
+      </div>
+      <div class="block__six">
+        <span class="block__number">6</span>
+      </div>
```

Comment:
why do you need wrappers around your spans? that seems to be an overcomplication